qa/TL0_FW_iterators/test_pytorch.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -e
 # used pip packages
 
-
-pip_packages="nose numpy opencv-python torch torchvision"
+# TODO(janton): remove explicit pillow version installation when torch fixes the issue with PILLOW_VERSION not being defined
+pip_packages="pillow==6.2.2 nose numpy opencv-python torch torchvision"
```

Comment:
Why not to add pillow to qa/setup_packages.py and have the version fixed there (only in one place).

Reply:
Mainly because we don't depend on pillow explicitly, having here `pillow==6.2.2` together with the TODO message clearly states that we are doing this because of `torchvision` while having only `pillow` hides that

It's not a problem for me to change it if you think it's better that way